### ISSUE TITLE
1111111

### ISSUE THREAD

**Issue by Stick-To** [Stick-To/fc_net-with-matlab#1]
import tensorflow as tf
import numpy as np
lr_images = tf.placeholder(tf.float32, shape=[1, 8, 8, 4])
output = tf.layers.conv2d(lr_images, 4, 1, padding='same')
output = tf.nn.relu(output)
output = tf.nn.depth_to_space(output, block_size=2)
sess = tf.InteractiveSession()
sess.run(tf.global_variables_initializer())
for i in range(100):
    a = sess.run(output, feed_dict={lr_images: np.random.rand(1, 8, 8, 4)})
    print(a.shape)

**Comment by Stick-To** [comment]
import tensorflow as tf
import numpy as np
lr_images_ = tf.placeholder(tf.float32, shape=[None, 8, 8, 4])
hr_images_ = tf.placeholder(tf.float32, shape=[None, 16, 16, 1])
output = tf.layers.conv2d(lr_images_, 4, 1, padding='same')
output = tf.nn.relu(output)
output = tf.nn.depth_to_space(output, block_size=2)

loss = tf.reduce_mean(output - hr_images_)

opti  = tf.train.AdamOptimizer(0.0002)
train_step = opti.minimize(loss)

sess = tf.InteractiveSession()
sess.run(tf.global_variables_initializer())

for i in range(100):
    lr_images = np.random.rand(1, 8, 8, 4)
    hr_images = np.random.rand(1, 16, 16, 1)
    _, a = sess.run([train_step, loss], feed_dict={lr_images_: lr_images,
                                  hr_images_:hr_images })
    print(a)
